Vostok.Metrics.System/Process/NativeMetricsCollector_Windows.cs

```diff
@@ -84,7 +85,10 @@ private void CollectCpuUtilization(CurrentProcessMetrics metrics)
                 var systemTime = systemKernel.ToUInt64() + systemUser.ToUInt64();
                 var processTime = processKernel.ToUInt64() + processUser.ToUInt64();
 
-                cpuCollector.Collect(metrics, systemTime, processTime);
+                // note: Environment.ProcessorCount from NET6+ respects process affinity and the job object's hard limit on CPU utilization
```

Comment:
`Environment.ProcessorCount` is used in some other places in this repo. Is it correct to leave it as-is?